includes/civicrm.basepage.php

```diff
@@ -332,48 +332,58 @@ public function basepage_handler($wp) {
 
         global $post;
 
-        // Skip if this not the Base Page.
-        if ($basepage->ID !== $post->ID) {
-          continue;
-        }
-        else {
-          $basepage_found = TRUE;
-        }
-
-        // Set context.
-        $this->civi->civicrm_context_set('basepage');
-
-        // Start buffering.
-        ob_start();
-        // Now, instead of echoing, base page output ends up in buffer.
-        $this->civi->invoke();
-        // Save the output and flush the buffer.
-        $this->basepage_markup = ob_get_clean();
-
-        /*
-         * The following logic is in response to some of the complexities of how
-         * titles are handled in WordPress, particularly when there are SEO
-         * plugins present that modify the title for Open Graph purposes. There
-         * have also been issues with the default WordPress themes, which modify
-         * the title using the 'wp_title' filter.
+        /**
+         * Skip if this not the Base Page, but allow "Base Page mode" to be forced.
+         *
+         * Return TRUE to force CiviCRM to render a Post of Page as if on the Base Page.
          *
-         * First, we try and set the title of the page object, which will work
-         * if the loop is not run subsequently and if there are no additional
-         * filters on the title.
+         * @since 5.35
```

Comment:
as we are targeting master this should be 5.36

Reply:
Thanks for pointing this out @kcristiano I'll tweak when we've reached consensus 👍

Reply:
Fixed in 0b557e02a36a08a2b51d92e027874e196e8771c8